.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+   "cSpell.words": [
+      "timetrackerappdata",
+      "typeorm"
+   ]
+}
```

Comment:
(comment): Some teams will have a style guide that requires all files to end in a newline character.